applications/nrf_desktop/src/modules/Kconfig.usb_state

```diff
@@ -96,9 +96,16 @@ config DESKTOP_USB_STACK_NEXT
 	bool "USB next stack [EXPERIMENTAL]"
 	select EXPERIMENTAL
 	select USB_DEVICE_STACK_NEXT
+	imply SOC_FLASH_NRF_PARTIAL_ERASE
```

Comment:
What is the impact of this `imply` for devices without Flash memory (like nRF54L15 or nRF54H20)?

Reply:
It actually does nothing